packages/web-frontend/src/containers/mobile/MapOverlayLegend.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+import Button from '@material-ui/core/Button';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import { Legend } from '@tupaia/ui-components/lib/map';
```

Comment:
@tcaiger actually set it up so that there's code splitting within `@tupaia/ui-components`, so that theoretically the bundle size of any consuming package could be smaller by e.g. only importing the map stuff if it's needed